tests_and_analysis/test/utils.py

```diff
@@ -202,3 +202,40 @@ def check_json_metadata(json_file: str, class_name: str):
         data = json.loads(f.read())
     assert data['__euphonic_class__'] == class_name
     assert data['__euphonic_version__'] == __version__
+
+def sum_at_degenerate_modes(values_to_sum, frequencies, TOL=0.05):
```

Comment:
What's with the all-caps?

If we're using TOL to indicate some kind of global constant then it would make more sense to define at a higher scope. If it's just a function argument, I don't see a clash with `tolerance`?

Reply:
Good point, this was left over from the previous version of the function which I wrote quite a while ago, I'll change it.